zebra-state/src/service/finalized_state/disk_format/upgrade/add_subtrees.rs

```diff
@@ -434,22 +891,18 @@ fn write_sapling_subtree(
         .write_batch(batch)
         .expect("writing sapling note commitment subtrees should always succeed.");
 
-    if index.0 % 100 == 0 {
-        info!(?height, index = ?index.0, "calculated and added sapling subtree");
+    if subtree.index.0 % 100 == 0 {
+        info!(end_height = ?subtree.end, index = ?subtree.index.0, "calculated and added sapling subtree");
     }
     // This log happens about once per second on recent machines with SSD disks.
-    debug!(?height, index = ?index.0, ?node, "calculated and added sapling subtree");
+    debug!(end_height = ?subtree.end, index = ?subtree.index.0, "calculated and added sapling subtree");
 }
 
 /// Writes a Orchard note commitment subtree to `upgrade_db`.
```

Comment:
###### Optional Suggestion

```suggestion
/// Writes an Orchard note commitment subtree to `upgrade_db`.
```